templates/rdfLibraryDependent/java/commonsRdfServiceLoader/vocab.hbs

```diff
@@ -49,7 +55,7 @@ public class {{vocabNameUpperCase}} {
     /**
      *{{#if comment}} {{{comment}}}{{/if}}{{#if termDescription}}
      *
-     * {{termDescription}}{{/if}}{{#if seeAlsos}}
+     * {{{termDescription}}}{{/if}}{{#if seeAlsos}}
```

Comment:
Why the additional pair of curly brackets ?

Reply:
It's a Handlebars syntax thing that says '_Don't encode the characters in this string_', which is what is needed here (as the single quote character in descriptive string was getting encoded as #0x67 or something without this).